apps/system/utils/Kconfig

```diff
@@ -12,6 +12,21 @@ config SYSTEM_CMDS
 		system commands. Because they can be used without TASH by function calling.
 		Let's enable system commands at each configuration.
 
+config SECURITY_LEVEL_CMDS
+	bool "Security level commands"
+	default y
+        depends on TASH && SECURITY_LEVEL
+	select SECURITY_API
+	select SECURITY_SS
```

Comment:
1. Let's use the same way for indentation.  You use spaces in line 18, but use a tab for other lines.
2. When do you use `depends on` and `select`?